scripts/test-e2e

```diff
@@ -2,36 +2,58 @@
 
 set -ueo pipefail
 
+cd $(dirname "$0")
+
+server="../dist/toxiproxy-server"
+
+benchmark() {
+  go test -bench=.. ../test/e2e -v
+}
+
+cli() {
+  ../dist/toxiproxy-cli $* 2>&1 | sed -e 's/^/[client] /'
```

Comment:
```suggestion
  ../dist/toxiproxy-cli "$@" 2>&1 | sed -e 's/^/[client] /'
```

This is to ensure expanded argv is still quoted if arguments include special characters.

Reply:
thank you, fixed.